package.json

```diff
@@ -24,7 +25,8 @@
     "mocha": "^6.2.2",
     "nyc": "^14.1.1",
     "ot-json1": "^1.0.1",
-    "sharedb": "^1.0.0-beta"
+    "sharedb": "^1.0.0",
+    "sinon": "^11.1.2"
```

Comment:
I don't see sinon used in this PR, it can be removed.

Reply:
Oh right I forgot to mention this... This was one of two issues when trying to run all tests. The other is that another test is failing inside the `sharedb` test code. I assume sinon is used by those tests, and somehow is not being imported from there? 

Failing test:
```
1) db
       client query subscribe
         passing agent.custom to the DB adapter
           sends agentCustom to the db's getSnapshot call:

      Uncaught AssertionError: expected 3 to equal 2
      + expected - actual

      -3
      +2
      
      at Query.<anonymous> (node_modules/sharedb/test/client/query-subscribe.js:472:47)
      at Query._handleDiff (node_modules/sharedb/lib/client/query.js:177:14)
      at Connection.handleMessage (node_modules/sharedb/lib/client/connection.js:220:31)
      at StreamSocket.socket.onmessage (node_modules/sharedb/lib/client/connection.js:142:18)
      at /Users/cburger/lever/sharedb-mingo-memory/node_modules/sharedb/lib/stream-socket.js:60:12
      at process._tickCallback (internal/process/next_tick.js:61:11)
```

Sinon issue:
```
npm run test

> sharedb-mingo-memory@1.1.4 test /Users/cburger/lever/sharedb-mingo-memory
> mocha

/Users/cburger/lever/sharedb-mingo-memory/node_modules/yargs/yargs.js:1172
      else throw err
           ^

Error: Cannot find module 'sinon'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/Users/cburger/lever/sharedb-mingo-memory/node_modules/sharedb/test/client/query-subscribe.js:4:13)
 ```